js/ToDo.js

```diff
@@ -0,0 +1,53 @@
+const list = {
+    "create a task": "In Progress",
+    // "make a bed": "Done",
+    "write a post": "To Do",
+}
+
+function changeStatus() {
+    list["write a post"] = "Done"
+}
+function addTask() {
+    list['have a walk'] = "To Do"
+}
+function deleteTask() {
+    delete list['have a walk']
+}
```

Comment:
Почти, но не совсем. Не вижу, где используются эти функции. Пока что они способны выполнять одну задачу. Но с теми же успехами можно было написать и вот так

```suggestion
    list["write a post"] = "Done"
    list['have a walk'] = "To Do"
    delete list['have a walk']
```

Сама идея TODO листа в том, что им нужно пользоваться. Например, мы хотим добавить задачу в туду. Или мы хотим поменять ее статус. Ну, и конечно же нам лень ее делать и мы хотим ее удалить. Подумай над тем, как сделать функции функциональными, чтобы их можно было использовать

Reply:
Попробуй запустить файл node js/ToDo.js, при этом:

1. Выведи в консоль исходный туду лист
2. Измени его с помощью функций (добавить новые задачи, изменить их статус, удалить текущие)
3. Выведи что получилось

Результат работы консоли можешь оставить здесь в комментарий, когда скинешь на повторное ревью